src/algorithms/mazeGenerators/Maze.java

```diff
@@ -13,19 +20,21 @@
  * mazeArr - int array for the maze's presentation
  * posArr - Array of positions holding each position at it's place.
  */
-public class Maze {
+public class Maze implements Serializable{
```

Comment:
is the serializable necessary?

Reply:
Yes, if an object is not serializable it can't be sent as an Object stream